standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -1937,24 +1937,38 @@ public List<Partition> dropPartitions(String catName, String dbName, String tblN
                                         PartitionDropOptions options) throws TException {
     RequestPartsSpec rps = new RequestPartsSpec();
     List<DropPartitionsExpr> exprs = new ArrayList<>(partExprs.size());
+    for (Pair<Integer, byte[]> partExpr : partExprs) {
```

Comment:
This was moved to `ThriftHiveMetaStoreClient`, no wonder i wasn't able to find link with SessionHiveMetaStoreClient.
This PR needs rebase.